src/responses.rs

```diff
@@ -654,6 +654,17 @@ impl fmt::Display for MessageProperties {
     }
 }
 
+#[derive(Debug, Deserialize, Clone, Eq, PartialEq)]
+pub struct Overview {
+    pub cluster_name: String,
+    pub erlang_full_version: String,
+    pub erlang_version: String,
+    pub management_version: String,
```

Comment:
This will never be different from `rabbitmq_version`, I don't think we need it (neither do we need it in the API response but it will take a deprecation period before it can be removed).